lib/pages/chat/input_bar/input_bar_shortcut.dart

```diff
@@ -16,13 +16,16 @@ class InputBarShortcuts extends StatelessWidget {
 
   final FocusSuggestionController? focusSuggestionController;
 
+  final ScrollController scrollController;
+
   InputBarShortcuts({
     super.key,
-    required this.child,
+    required this.scrollController,
     this.room,
     this.controller,
     this.onEnter,
     this.focusSuggestionController,
+    required this.child,
```

Comment:
IMO, all `required` properties should be right after key then the optional properties

Reply:
every flutter widget, put the child/children as the last element in parameter lists